core/src/main/java/org/jboss/logmanager/LoggerNode.java

```diff
@@ -487,19 +487,41 @@ private static boolean isLoggable(final LoggerNode loggerNode, final ExtLogRecor
         return !(filter != null && !filter.isLoggable(record)) && (!loggerNode.useParentFilter || isLoggable(loggerNode.getParent(), record));
     }
 
-    // GC
+    Enumeration<String> getLoggerNames() {
```

Comment:
I thought about that, but I thought that buffering the whole set of logger names before returning it would be a bit of a waste of memory (n versus log(n)).

Reply:
That makes sense. It's not a public API anyway so it doesn't matter.